grakn-engine/src/main/java/ai/grakn/engine/loader/MutatorTask.java

```diff
@@ -50,10 +55,10 @@
     private final QueryBuilder builder = Graql.withoutGraph().infer(false);
 
     @Override
-    public boolean start(Consumer<TaskCheckpoint> saveCheckpoint, TaskConfiguration configuration) {
+    public boolean start(Consumer<TaskCheckpoint> saveCheckpoint, TaskConfiguration configuration, BiConsumer<TaskState, TaskConfiguration> taskSubmitter) {
```

Comment:
Will fix that in another PR. Next week I have a bunch of cleanup and bug tasks to attend to. This will be one of them.